lib/rules/no-extra-parens.js

```diff
@@ -431,6 +435,10 @@ module.exports = {
                     return;
                 }
 
+                if (node.body.type === "ConditionalExpression" && ENFORCE_FOR_ARROW_CONDITIONALS) {
+                    return;
+                }
```

Comment:
I think the option is backwards. When `enforceForArrowConditionals` is *true*, the conditional expressions *should* be checked. When `enforceForArrowConditionals` is *false*, conditional expressions *should not* be checked.